CONTRIBUTING.md

```diff
@@ -38,68 +42,46 @@ Working on your first Pull Request? You can learn how from this *free* series [H
 - [Set Up freeCodeCamp](#set-up-freecodecamp)
 
 ### [Make Changes](#make-changes)
+
 - [Unpack and Repack](#unpack-and-repack)
 - [Challenge Template](#challenge-template)
 - [Run The Test Suite](#run-the-test-suite)
 
 ### Submit
+
 - [Creating a Pull Request](#creating-a-pull-request)
 - [Common Steps](#common-steps)
 - [How We Review and Merge Pull Requests](#how-we-review-and-merge-pull-requests)
 - [How We Close Stale Issues](#how-we-close-stale-issues)
 - [Next Steps](#next-steps)
 - [Other Resources](#other-resources)
 
-## Setup
-### Prerequisites
-
-| Prerequisite                                | Version |
-| ------------------------------------------- | ------- |
-| [MongoDB Community Server](https://docs.mongodb.com/manual/administration/install-community/) | `~ ^3`  |
-| [MailHog](https://github.com/mailhog/MailHog) | `~ ^1` |
-| [Node.js](http://nodejs.org)                | `~ ^8.9.3`  |
-| npm (comes with Node)                       | `~ ^5`  |
-
-> _Updating to the latest releases is recommended_.
-
-If Node.js or MongoDB is already installed on your machine, run the following commands to validate the versions:
-
-```shell
-node -v
-mongo --version
-```
-
-To check your MongoDB version on Windows, you have to locate the installation directory. It is probably located at something like `C:\Program Files\MongoDB\Server\3.4\` where 3.4 is your version number.
-
-If your versions are lower than the prerequisite versions, you should update.
-
-Platform-specific guides to setting up a development environment:
-- [How to clone and setup the freeCodeCamp website on a Windows pc](https://forum.freecodecamp.org/t/how-to-clone-and-setup-the-free-code-camp-website-on-a-windows-pc/19366)
-- [How to Clone and Setup the freeCodeCamp Website on a Mac](https://forum.freecodecamp.org/t/how-to-clone-and-setup-the-freecodecamp-website-on-a-mac/78450)
-
-### Forking the Project
+### Getting a copy the freeCodeCamp Project
 
 #### Setting Up Your System
 
-1. Install [Git](https://git-scm.com/) or your favorite Git client.
-2. (Optional) [Setup an SSH Key](https://help.github.com/articles/generating-an-ssh-key/) for GitHub.
+1. [Install Git](https://git-scm.com/) or your favorite Git client.
+2. [Setup a SSH Key](https://help.github.com/articles/generating-an-ssh-key/) for GitHub. (Optional)
 
 #### Forking freeCodeCamp
 
-1. Go to the top level freeCodeCamp repository: <https://github.com/freeCodeCamp/freeCodeCamp>
-2. Click the "Fork" Button in the upper right hand corner of the interface ([More Details Here](https://help.github.com/articles/fork-a-repo/))
-3. After the repository (repo) has been forked, you will be taken to your copy of the freeCodeCamp repo at <https://github.com/yourUsername/freeCodeCamp>
+1. Go to the top level freeCodeCamp repository: [`https://github.com/freeCodeCamp/freeCodeCamp`](https://github.com/freeCodeCamp/freeCodeCamp)
```

Comment:
Hyphenate ["top-level"](https://english.stackexchange.com/questions/45945/how-does-the-hyphen-change-the-meaning-in-expressions-like-high-performance-an)? Or reword to "Go to the main freeCodeCamp repository"?

Reply:
"Setup **a** SSH Key" should be "Setup **an** SSH Key"